### ISSUE TITLE
[bug] [ios] Unlocking a vault shows the splash screen

### ISSUE THREAD

**Issue by lfom** [buttercup/buttercup-mobile#308]
I have only one vault that is accessed via WebDAV, when opening it with FaceID on iOS most of the times the app shows the splash screen preventing any user interaction. Workaround: close the app or just go to another app or the home screen and then back to Buttercup. It is annoying nevertheless.

**Comment by perry-mitchell** [comment]
Never seen this. Can you include a screen recording?

**Comment by lfom** [comment]
Yes, it's not happening anymore here either. Maybe it was related to pCloud stopping to support WebDAV for the free accounts, I haven't had the problem since I moved to NextCloud. Cheers

**Comment by lfom** [comment]
Hello! I am able to reproduce the issue now using Buttercup 2.4.2 on iOS every time I follow these steps:

1. Make sure you have a vault with Biometric Unlock enabled
2. Unlock the vault then go back to the main screen
3. Use Lock All Vaults
4. Try to open the same vault again
5. It will unlock it using FaceID and then get stuck in the splash screen

It is not really stuck tho: I while ago I have found out that you can swipe it right to go back to the main screen again.
